src/pages/products/sgid/transportation/mile-reference-posts.astro

```diff
@@ -0,0 +1,141 @@
+  <Metadata slot="metadata" {...page} />
+
+  <Fragment slot="summary">
+    <p>
+      The Mile Reference Posts dataset contains the physical location of milepost signs as they are located along the
+      roads. This collection technique is different from <ExternalLink
+        href="https://opendata.gis.utah.gov/datasets/uplan::mile-point-measure-tenth"
+        >linear mile point measures</ExternalLink
+      > that are derived from a linear reference system (LRS). The advantage of this collection method is that the mile reference
+      posts stay in roughly the same place over time and do not change when other sections of the road, or LRS, are realigned.
+    </p>
+  </Fragment>
+
+  <Section title="Use the data" slot="downloads" titlePosition="center">
+    <HubDownloads {...page.hub} />
+  </Section>
+
+  <Fragment slot="description">
+    <p>
+      Mile reference post locations are derived from images captured by <ExternalLink href="https://www.mandli.com/"
+        >Mandli Communications</ExternalLink
+      >, using on-the-ground asset collection technology. These locations were then aligned to UDOT's linear referencing
+      system.
+    </p>
+    <p>
+      This data will be updated through on-the-ground collection every 2 years under the current contract. The data
+      served on this page was captured in 2021 and reflects what was on the ground at that time.
+    </p>
+    <p>
+      The Mile Reference Posts is an <ExternalLink
+        href="https://www.esri.com/en-us/arcgis/products/arcgis-roads-highways/overview"
+        >ESRI Roads and Highways Event</ExternalLink
+      > layer and supports the <ExternalLink href="https://maps.udot.utah.gov/wadocuments/Apps/Stationing/"
+        >RP Stationing App</ExternalLink
+      >.
+    </p>
+    <p>
+      Please reach out to <Contacts contactKey="udot" subject="gis.utah.gov mile reference post layer inquiry" inline />
+      with questions about this data layer.
+    </p>
+    <h3>Mile Reference Posts vs Linear Mile Point Measures</h3>
+    <h4>Mile Reference Posts</h4>
+    <p>
+      Mile Reference Posts are signs located in physical locations. They are helpful for navigation but may not reflect
+      the linear referencing measurement.
+    </p>
+    <h4>Linear Mile Point Measures</h4>
+    <p>
+      <ExternalLink href="https://opendata.gis.utah.gov/datasets/uplan::mile-point-measure-tenth"
+        >Linear Mile Point Measures</ExternalLink
+      > are calculated based on geometric lengths of data and are spatially accurate. They may or may not align with actual
+      Mile Reference Posts and are updated every time a change is made in the spatial data to reflect updates in the physical
+      location and length of the roadway.
+    </p>
+    <p>
+      For more information on the difference between mile reference posts and linear measures, please refer to <ExternalLink
+        href="https://udot.utah.gov/connect">UDOT's webpage</ExternalLink
```

Comment:
**polish**: i added this link since it was empty, did you want to put something else in here?

Reply:
good catch.

Reply:
fixed